### PR TITLE
Accessibility improvements 

### PR DESCRIPTION
**Pull request by sabrinaharris**
Accessibility improvements to make links more descriptive and remove bold (unless it's used for a UI element, which is OK).

### CODE REVIEW

File: source/legacy/build-ms/respond-to-matching-requests/index.html.md.erb [hunk 1]
```diff
@@ -13,7 +13,7 @@ Your service must respond to JSON matching requests from the Matching Service Ad
 
 The MSA sends one matching request for both Cycle 0 and Cycle 1 to your local matching service. Below is a formatted example for a GOV.UK Verify identity using the [msa-universal-json-schema].
 
-The matching request for a [European identity][msa-eu-identities], will be formatted according to the [universal JSON schema][msa-universal-json-schema]. If you haven't enabled European identities in your MSA configuration, the matching request will be formatted using [the legacy JSON schema][msa-legacy-json-schema].
+The matching request for a European identity will be formatted according to the [universal JSON schema][msa-universal-json-schema]. If you haven't enabled European identities in your MSA configuration, the matching request will be formatted using [the legacy JSON schema][msa-legacy-json-schema]. [Find out more about European identities and eIDAS][msa-eu-identities].
```
Comment (Wynndow) [inline source/legacy/build-ms/respond-to-matching-requests/index.html.md.erb:16]: Is it worth updating line 14 to match the link text here? Currently is says "`Below is a formatted example for a GOV.UK Verify identity using the [msa-universal-json-schema].`" Would be nicer if it was "`Below is a formatted example for a GOV.UK Verify identity using the [universal JSON schema][msa-universal-json-schema].`"

I know this isn't stuff you were changing but I just spotted it!

**Review by Wynndow — APPROVED** [review]
One small thing, feel free to ignore it!

File: source/legacy/build-ms/respond-to-matching-requests/index.html.md.erb (hunk 1 above)
Comment (sabrinaharris) [reply source/legacy/build-ms/respond-to-matching-requests/index.html.md.erb:16]: Great catch, I'll update this too. Thanks!

**Review by Wynndow — APPROVED** [review]
🚀

**Pull request merged**